src/applications/claims-status/components/WhatWereDoing.jsx

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router';
+
+const statusMap = {
+  CLAIM_RECEIVED: 'Step 1 of 5: Claim received',
+  INITIAL_REVIEW: 'Step 2 of 5: Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Step 3 of 5: Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Step 4 of 5: Preparation for notification',
+  COMPLETE: 'Step 5 of 5: Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
+}
+
+function WhatWereDoing({ claim }) {
```

Comment:
Can we change the name of this to `WhatWeAreDoing`? It bothers me that it reads 'What were doing'

Reply:
haha sure I can rename it if needed, I was just going off what the section name was and I didnt think it was appropriate to have a ` in it.

Reply:
Renamed!